### DISCUSSION TITLE
I did a service using Unicorn, my experiences

### DISCUSSION THREAD

**Discussion by Rip4568** [django-commons/django-unicorn#492] [Show and tell]
Hello people, probably none of you know me but I want to state right away that this project called django-unicorn was by far the best django frontend framework I have seen, my confidence is such that I was asked to develop a web application for a pastry shop.
https://pastelaria-jj.herokuapp.com/
(unfortunately I can't make the github available because it is a private project)
I honestly enjoyed all the development using unicorn, with the exception of having some problems with the modal where every time a variable changed the modal disappeared.
Another problem is the child components that I couldn't make work at all, I ended up giving up on making it work and focusing on the most important thing which is the development of the site, unfortunately this decision made me commit the famous mistake of: "don't repeat yourself", that is: I had to copy and paste some functions that were in other views.
SUMMARY: Great project and I will keep on using it, I wish strength to the creator and congratulations for the great project.

**Comment by adamghill** [comment]
@Rip4568 I'm glad your experience went well! I hope I can fix the issues you ran into over time. 🦄
